pkg/services/store/http.go

```diff
@@ -27,26 +31,95 @@ func ProvideHTTPService(store StorageService) HTTPStorageService {
 	}
 }
 
+func UploadErrorToStatusCode(err error) int {
+	if errors.Is(err, ErrUploadFeatureDisabled) {
+		return 404
+	}
+
+	if errors.Is(err, ErrUnsupportedFolder) {
+		return 400
+	}
+
+	if errors.Is(err, ErrFileTooBig) {
+		return 400
+	}
+
+	if errors.Is(err, ErrInvalidPath) {
+		return 400
+	}
+
+	if errors.Is(err, ErrInvalidFileType) {
+		return 400
+	}
+
+	if errors.Is(err, ErrFileAlreadyExists) {
+		return 400
+	}
+
+	return 500
```

Comment:
Oh my Go

Reply:
I think you can use a switch statement here...  slightly more go idiomatic 🤷‍♂️ 

https://medium.com/@sebdah/go-best-practices-error-handling-2d15e1f0c5ee

Reply:
> Oh my Go

Gorror! Refactored to a switch stmnt